terraform/modules/hub/hub_config.tf

```diff
@@ -18,14 +18,31 @@ module "config_ecs_asg" {
   logit_elasticsearch_url = "${var.logit_elasticsearch_url}"
 }
 
+locals {
+  config_location_blocks = <<-LOCATIONS
+  location = /healthcheck {
```

Comment:
can we remove the healthcheck and use /service-status for config?

Reply:
or rather /service status is used but this contains a healthcheck location for no reason

Reply:
Thanks for spotting. It looks like My rebase error has brought the old code back. I will remove it.

Reply:
:)

Reply:
I have resolved the rebase error. The old code is no longer present.